assets/css/v2/on_bus/header.scss

```diff
@@ -0,0 +1,16 @@
+@use "CSS/colors";
+
+.departures-header {
+  display: flex;
+  align-items: center;
+  height: 108px;
+  margin-left: 26px;
+  font-family: Inter;
+  font-size: 47px;
+  font-style: normal;
+  font-weight: 500;
+  line-height: normal;
+  color: colors.$cool-black-30;
+  text-align: left;
```

Comment:
```css
font-style: normal;
line-height: normal;
text-align: left;
```

Are these all required to "undo" something else being set at a higher level? I actually see a number of instances of `font-style: normal` in this changeset, but no instances of a `font-style` with any other value.

Reply:
Removed the `font-style: normal` from throughout these changes! Also forgot that `text-align: left` is the default so that shouldn't be needed.